examples/jsm/nodes/accessors/Object3DNode.js

```diff
@@ -84,19 +83,19 @@ class Object3DNode extends Node {
 
 		if ( scope === Object3DNode.WORLD_MATRIX || scope === Object3DNode.VIEW_MATRIX ) {
 
-			this._inputNode = new Matrix4Node( /*null*/ );
+			this._uniformNode = new UniformNode( null, 'mat4' );
 
 		} else if ( scope === Object3DNode.NORMAL_MATRIX ) {
 
-			this._inputNode = new Matrix3Node( /*null*/ );
+			this._uniformNode = new UniformNode( null, 'mat3' );
 
 		} else if ( scope === Object3DNode.POSITION || scope === Object3DNode.VIEW_POSITION ) {
 
-			this._inputNode = new Vector3Node();
+			this._uniformNode = new UniformNode( new Vector3(), 'vec3' );
```

Comment:
I think that we can remove the second parameter here.

Reply:
Done.